src/feature/featurelistcallback.h

```diff
@@ -69,6 +69,13 @@ bool FeatureCallback_captivePortal() {
 #endif
 }
 
+bool FeatureCallback_inAppAuthentication() {
+  if (Constants::inProduction() || FeatureCallback_iosOrAndroid()) {
```

Comment:
Try ifdef MZ_DUMMY -> return true. 

Currently the functional tests all assume we do in app auth. we would need to roll back those patches 😅